flintrock/flintrock.py

```diff
@@ -236,6 +236,10 @@ def cli(cli_context, config, provider):
 @click.option('--ec2-user-data',
               type=click.File(mode='r', encoding='utf-8'),
               help="Path to EC2 user data script that will run on instance launch.")
+@click.option('--ec2-tag', 'ec2_tags',
+              multiple=True,
+              help="Additional tags (Key,Value pairs) to assign to the instances. "
```

Comment:
Nitpick: I think this would be slightly clearer if it were written as: "Additional tags (e.g. 'Key,Value') to assign to the instances. "